packages/config-loader/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "build": "tsc",
-    "test:node": "mocha test/**/*.test.js --reporter dot",
+    "test:node": "mocha test/**/*.test.js --reporter dot --no-experimental-detect-module",
```

Comment:
Is this _roughly the same_ as adding a `"type": "commonjs"` entry?

Reply:
I'm not 100% sure where this package fits in the ecosystem, but doing so would prevent any consumers from having a similar issue, right?

Reply:
@Westbrook The package is used by `@web/dev-server` and `@web/test-runner` to load respective configs. 

Updated to use `"type": "commonjs"` and added a changeset. I think it's fine to use patch release for this.